index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "moleculer-apollo-server" {
+	import { ServiceSchema } from "moleculer";
+	import { Config } from "apollo-server-core";
```

Comment:
This is unfortunate, although I'm not sure what (if anything) can be done about it.
One of these imports is leading to an implicit any from `node-fetch`:
```
node_modules/apollo-env/lib/fetch/fetch.d.ts:6:195 - error TS7016: Could not find a declaration file for module 'node-fetch'. 'C:/Users/mck32/Projects/moleculer-services/node_modules/node-fetch/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/node-fetch` if it exists or add a new declaration (.d.ts) file containing `declare module 'node-fetch';`
```
There is a DefinitelyTyped definition for `node-fetch`, so its resolvable by installing that; it's just unfortunate because I don't think its actually required by anything being used in moleculer-apollo-server.

Reply:
I think, this is only for the dev one. I added `@types/node-fetch` to `devDependencies`